e2e/cli-plugins/run_test.go

```diff
@@ -194,3 +194,15 @@ func TestCliInitialized(t *testing.T) {
 	assert.Assert(t, res.Stdout() != "")
 	assert.Assert(t, is.Equal(res.Stderr(), ""))
 }
+
+// TestPluginErrorCode tests when the plugin return with a given exit status.
+// We want to verify that the exit status does not get output to stdout and also that we return the exit code.
```

Comment:
Given that first bit you might want to add
```go
assert.Assert(t, is.Equal(res.Stdout(), ""))
```
since `icmd.Expected` doesn't check anything when `Out == ""`?